browser/main/SideNav/SearchButton.styl

```diff
@@ -0,0 +1,55 @@
+.top-menu-search
+  navButtonColor()
+  position relative
+  margin-right 6px
+  top 3px
```

Comment:
You've perfectly aligned the search glass to bottom (see first screenshot below). But I think it would be visually better to move it slightly up.

![grafik](https://user-images.githubusercontent.com/3046542/57411291-6825b400-71ed-11e9-9421-facd885e941b.png)

The following is with 2px as top
![grafik](https://user-images.githubusercontent.com/3046542/57411361-a28f5100-71ed-11e9-9ec7-77eb9cc2a8f4.png)